client/src/pages/Income/Income.tsx

```diff
@@ -2,47 +2,31 @@ import React, { useState, useEffect } from "react";
 import { Container } from "./Income.styles";
 import { IncomeTracker, IncomeAdder } from "../../components";
 import { AnimatePresence } from "framer-motion";
+import { getAllIncomeItems } from "../../API/IncomeMethods";
 
 export interface IncomeData {
-  id: number;
+  _id: string;
   title: string;
   amount: number;
+  type: "Income" | "Expense";
 }
 
-// Static dummy DATA for example list
-const dummyIncomeData: Array<IncomeData> = [
-  {
-    id: 1,
-    title: "Weekly Check",
-    amount: 1200,
-  },
-  {
-    id: 2,
-    title: "Tax Return",
-    amount: 3500,
-  },
-  {
-    id: 3,
-    title: "Gifted",
-    amount: 600,
-  },
-];
-
-const Income = () => {
+const Income: React.FC = () => {
   const [displayAdder, setDisplayAdder] = useState<boolean>(false);
-  const [filteredIncomeData, setFilteredIncomeData] =
-    useState<Array<IncomeData>>(dummyIncomeData);
+  const [change, setChange] = useState<boolean>(false);
```

Comment:
Do the same things as we did in the expense section.  Honestly, after merging this ticket.  You may want to go ahead and refactor the incomes/expenses code to be the same code.  These PRs are huge and you are basically having to copy everything from file to file.  Not your fault, you didn't do anything wrong, but we can fix this to save us time moving forward.